### DISCUSSION TITLE
Zitadel Login Time Taken

### DISCUSSION THREAD

**Discussion by santhosh-stitch** [zitadel/zitadel#6459] [Q&A]
Hi,

Please help us on the following. I tried to log in to the zitadel self-host, but it required 7.7 MB for the resources download.

<img width="713" alt="Pasted Graphic" src="https://github.com/zitadel/zitadel/assets/104059308/96a02434-342a-4107-af5d-0562dfae662e">

The loading of the resources is taking longer.
Is this customizable in any way to speed up the loading process?

**Comment by mffap** [comment]
@santhosh-stitch did you login to an application or is the 7.7mb also including loading Console?
What version are you using? Did you use an image, or did you built it?

**Reply by santhosh-stitch** [reply]
@mffap It's a Zitadel custom user interface console.

**Comment by mffap** [comment]
> @mffap It's a Zitadel custom user interface console.

Sorry I don't know what you mean by that. 
It looks like you load the Console, then redirect to login. 
I don't see if you load multiple pages or what you are loading. Would be useful to know to scope the issue.

Can you post more detailed info about assets loaded, path, and size?

Also please let me know wich version you are using.
